src/Makefile

```diff
@@ -10,14 +10,17 @@ CC=gcc
 CFLAGS=-I$(IDIR) -std=c99 -Wall
 ODIR=obj
 EXE=bin
-
+TGDIR=../testing/GenFiles/
 
 _DEPS = defines.h trace.h
 DEPS = $(patsubst %,$(IDIR)/%,$(_DEPS))
 
 _OBJ = main.o trace.o
 OBJ = $(patsubst %,$(ODIR)/%,$(_OBJ))
 
+_TEST = TestGen.o
```

Comment:
Uh... Genuinely not 100% on anything I added here; just tried to imitate what was going on with the others. Please correct as needed.